sdk/src/main/java/com/hedera/hashgraph/sdk/TokenRejectFlow.java

```diff
@@ -0,0 +1,354 @@
+        return new TransactionReceiptQuery()
+            .setNodeAccountIds(Collections.singletonList(response.nodeId))
+            .setTransactionId(response.transactionId);
+    }
+
+    /**
+     * Execute the transactions in the flow with the passed in client.
+     *
+     * @param client the client with the transaction to execute
+     * @return the response
+     * @throws PrecheckStatusException when the precheck fails
+     * @throws TimeoutException        when the transaction times out
+     */
+    public TransactionResponse execute(Client client) throws PrecheckStatusException, TimeoutException {
+        return execute(client, client.getRequestTimeout());
+    }
+
+    /**
+     * Execute the transactions in the flow with the passed in client.
+     *
+     * @param client                the client with the transaction to execute
+     * @param timeoutPerTransaction The timeout after which each transaction's execution attempt will be cancelled.
+     * @return the response of TokenRejectTransaction
+     * @throws PrecheckStatusException when the precheck fails
+     * @throws TimeoutException        when the transaction times out
+     */
+    public TransactionResponse execute(Client client, Duration timeoutPerTransaction)
+        throws PrecheckStatusException, TimeoutException {
+        try {
+            var tokenRejectTxResponse = createTokenRejectTransaction().execute(client, timeoutPerTransaction);
+            tokenRejectTxResponse.getReceipt(client, timeoutPerTransaction);
+
+            var tokenDissociateTxResponse = createTokenDissociateTransaction().execute(client, timeoutPerTransaction);
+            tokenDissociateTxResponse.getReceipt(client, timeoutPerTransaction);
+
+            return tokenRejectTxResponse;
+        } catch (ReceiptStatusException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * Execute the transactions in the flow with the passed in client asynchronously.
+     *
+     * @param client the client with the transaction to execute
+     * @return the response
+     */
+    public CompletableFuture<TransactionResponse> executeAsync(Client client) {
+        return executeAsync(client, client.getRequestTimeout());
+    }
+
+    /**
+     * Execute the transactions in the flow with the passed in client asynchronously.
+     *
+     * @param client                the client with the transaction to execute
+     * @param timeoutPerTransaction The timeout after which each transaction's execution attempt will be cancelled.
+     * @return the response
+     */
+    public CompletableFuture<TransactionResponse> executeAsync(Client client, Duration timeoutPerTransaction) {
+        return createTokenRejectTransaction().executeAsync(client, timeoutPerTransaction)
+            .thenCompose(tokenRejectResponse -> createTransactionReceiptQuery(tokenRejectResponse).executeAsync(client, timeoutPerTransaction))
```

Comment:
Can we do  `tokenRejectResponse.getReceiptQuery()` here and remove the `createTransactionReceiptQuery` f

Reply:
Yup, done